patch/cisco-npu-disable-other-bars.patch

```diff
@@ -0,0 +1,90 @@
+From 2bc0b2e84866a7733fc05a34bd8a93e165db99cc Mon Sep 17 00:00:00 2001
+From: Madhava Reddy Siddareddygari <msiddare@cisco.com>
+Date: Tue, 9 Mar 2021 13:54:28 -0800
+Subject: [PATCH] NPU disable other bars
+
+    This is a port of IOS XR patch
+
+        Spitfire-pacific-disable-all-bars-other-than-0.patch
+
+    to Debian kernel. Modifications from IOS XR:
+
+    a. Only issue an INFO message for changed resources
+    b. Include the previous resource data in the INFO message
+    c. Prepare for the possibility of applying patch to
+       future NPU revisions.
+
+    Original patch message follows:
+
+    For Pacific, only bar 0 is valid. Hence disable all other bars as a work
+    around for spitfire
+
+    CDETS: CSCvi15029
+
+    Signed-off-by: Gaurav Gupta <gauragup@cisco.com>
+    Signed-off-by: Subha Keshavaraj <subkesha@cisco.com>
```

Comment:
It looks like you just copied the output from `git show`. Please create the patch with `git format-patch …`.